### DISCUSSION TITLE
How to change border Color Stepper ?

### DISCUSSION THREAD

**Discussion by d-strygwyr** [chakra-ui/chakra-ui#7762] [General]
![Screenshot 2023-06-14 023039](https://github.com/chakra-ui/chakra-ui/assets/49369600/7b9f888d-6f99-407c-bb71-0abb92e1661c)

if u asking why is the border black? "because I override the blue color "

so I want to ask,  how to change the border color from stepper without change from theme ?

**Comment by d-strygwyr** [comment]
idk this way is right or no, but it work,

i hope the next release can have some props for change the 

![image](https://github.com/chakra-ui/chakra-ui/assets/49369600/4c04e895-92c8-476a-950f-860ddfa96936)

**Comment by GoutamSamal9** [comment]
` <StepIndicator
                backgroundColor={step === "error" ? "red !important" : "green"}
              >
                <StepStatus
                  complete={
                    step === "error" ? <Icon icon={faClose} /> : <StepIcon />
                  }
                  incomplete={<StepNumber />}
                  active={<StepNumber />}
                />
              </StepIndicator>`
